x-pack/plugins/fleet/server/services/agents/agent_metrics.ts

```diff
@@ -9,6 +9,7 @@ import type { ElasticsearchClient } from '@kbn/core/server';
 
 import type { Agent } from '../../types';
 import { appContextService } from '../app_context';
+import { DATA_TIERS } from '../../routes/data_streams/get_data_streams_query_metadata';
```

Comment:
nit: the constant could be moved to a common place, importing from routes seems out of place here.

Reply:
moved under `common/constants`!